R/library_call_linter.R

```diff
@@ -0,0 +1,60 @@
+#' Library call linter
+#'
+#' Force library calls to all be at the top of the script.
+#'
+#' @examples
+#'  # will produce lints
+#'  lint(
+#'    text = c("library(dplyr)", "print('test')", "library(tidyr)"),
```

Comment:
We usually use multiline text instead of character vectors. We should stay consistent within the docs.
Can you refactor to 

```
text = "
  library(dplyr)
  print('test')
  library(tidyr)
"
```

or similar?

Reply:
Sure thing.  I'll cover this is my next commit!